--- openvpn-2.4.7.orig/configure.ac	2019-02-20 04:28:23.000000000 -0800
+++ openvpn-2.4.7/configure.ac	2021-03-03 22:48:46.663037694 -0800
@@ -54,7 +54,7 @@
                 awk '{split ($NF,a,"."); if (a[1] == 1 && a[2] >= 12) { print "serial-tests" }}'
     ])
 ])
-AM_INIT_AUTOMAKE(foreign serial_tests) dnl NB: Do not [quote] this parameter.
+AM_INIT_AUTOMAKE(foreign serial_tests subdir-objects) dnl NB: Do not [quote] this parameter.
 AC_CANONICAL_HOST
 AC_USE_SYSTEM_EXTENSIONS
 
